src/components/shared/Input/Input.js

```diff
@@ -1,19 +1,35 @@
 import React from 'react';
 import './Input.scss';
 
-const Input = ({label, handleChange, ...otherProps }) => {
+const Input = ({
+                   field,
+                   label,
+                   name,
+                   id,
+                   value,
+                   handleChange,
+                   form: {touched, errors},
+                   ...props
+               }) => {
     return (
         <div className='container'>
-            <input className='container__input' onChange={handleChange} {...otherProps} />
+            <input className='container__input' onChange={handleChange} {...props} id={id}
+                   type="text"
+                   {...field}
+                   {...props} />
             {label ? (
                 <label
                     className={`${
-                        otherProps.value.length ? 'container__shrink' : ''
+                        value.length ? 'container__shrink' : ''
                     }  container__label`}
                 >
                     {label}
                 </label>
             ) : null}
+            {touched[field.name] && errors[field.name] && (
+                <span className="container__error-class">
+                    {errors[field.name]}
+                </span>)}
```

Comment:
tutaj tez klasy do poprawy nie moze byc glowna klasa container mozna to zrobic np. input-container a pozniej custom-input, custom-input__label custom-input__error czy cos takiego do przemyslenia